docs/modules/ROOT/pages/cops_minitest.adoc

```diff
@@ -519,7 +519,7 @@ end
 | Yes
 | Yes
 | 0.7
-| 0.16
+| 0.17
```

Comment:
It looks like it contains unnecessary document update.

Reply:
Shouldn't the `Version Changed` be updated when adding new config option?

Reply:
Yeah, the default behavior with user impact does not change, so `VersionChanged` does not need to be added.

And FYI, this docs/modules/ROOT/pages/cops_minitest.adoc doc is auto-generated at the time of the next release. Please specify `VersionChanged: '<<next>>'` in config/default.yml if the value will be updated. (Not needed in this PR.)